ibm/mas_devops/roles/cp4d_db2wh_manage_hack/tasks/main.yml

```diff
@@ -82,6 +84,7 @@
 #   LOAD,CREATE_EXTERNAL_ROUTINE,QUIESCE_CONNECT,SECADM ON DATABASE TO USER {{ db2wh_username }}
 
 - name: Prepare db2 for use with Maximo TPAE
+  ignore_errors: yes
```

Comment:
Can we have a comment here explaining why it's okay to ignore errors please, for the benefit of our future selves?

Reply:
Fixed. Reason why i added it is because the same shell script returns a successful output for db2wh in cpd 3.5 but returns a fail for cpd 4.0 as it complains about granting privileges to self (db2inst1), although the overall db setup was successful, and manage/health can be installed just fine.

Reply:
Do we have an action item to address this oddity/make it pass even in cpd v4?